pkg/stubs/stubs.go

```diff
@@ -30,6 +34,9 @@ func Create(path string, testType config.FuzzTestType) error {
 	switch testType {
 	case config.CPP:
 		content = cppStub
+	case config.JAVA:
+		fileName := strings.TrimSuffix(filepath.Base(path), ".java")
+		content = []byte(strings.Replace(string(javaStub), "__CLASS_NAME__", fileName, 1))
```

Comment:
I would suggest `basename` instead of `fileName`:
```suggestion
		basename := strings.TrimSuffix(filepath.Base(path), ".java")
		content = []byte(strings.Replace(string(javaStub), "__CLASS_NAME__", basename, 1))
```